mhkit/tests/loads/test_loads.py

```diff
@@ -218,6 +218,13 @@ def test_shortterm_extreme(self):
             ste = loads.extreme.ste(t, data, t_st, method)
             assert_allclose(ste.cdf(x), cdf_1)
 
+    def test_automatic_threshold(self):
+        filename = "data_loads_hs.csv"
+        data = np.loadtxt(os.path.join(datadir, filename), delimiter=",")
+        years = 2.97
+        pct, threshold = loads.extreme.automatic_hs_threshold(data, years)
+        assert np.isclose(pct, 99.13)
```

Comment:
@cmichelenstrofer Apologies I was unclear when I typed my last message. The percentile is still returned as a fraction, so we should compare to 0.9913

```suggestion
        assert np.isclose(pct, 0.9913)